### ISSUE TITLE
Android Version Support (API LEVEL)

### ISSUE THREAD

**Issue by tjerkw** [Adyen/adyen-android#10]
What Androd Versions does this SDK support.

We (at Tiqets.com) integrated it, but the payment flow fails for Android VersionCode 19.

Can you explicitly mention which versions are supported?

**Comment by timon-langlotz** [comment]
Hi @tjerkw,
The minimum SDK version currently is 15 and will be 16 from the next update on.
In our test environment, we are now requiring TLS v.1.2. That's probably why it's failing for you. We will release an update this week where we fix the issue that network calls are not working on Android devices < 5.0 (21) in our test environment. The live environment should not be affected.

 I'm very sorry for the inconvenience and will update this issue once we have released our update.

Cheers,
Timon

**Comment by tjerkw** [comment]
Indeed it was an SSL issue on the test environment. Will test it on the live environment. Thanks for the quick feedback!

**Comment by tjerkw** [comment]
I'm opening this again, on the production Env payments also dont work. I'm getting a Payment Refused error on api level 19:

![screen shot 2017-10-09 at 15 00 25](https://user-images.githubusercontent.com/818865/31339396-065d7902-ad03-11e7-8142-2fce829f2263.png)

```
D/PaymentStateHandler: Processing payment.
D/PaymentStateHandler: processPayment()
D/PaymentStateHandler: The merchant application will handle UI while Checkout SDK is processing payment.
D/PaymentStateHandler: initiatePayment()
D/AsyncHttpClient: POST request for url: https://checkoutshopper-live.adyen.com/checkoutshopper/services/PaymentInitiation/v1/initiate
D/AsyncHttpClient: POST data: {"paymentDetails":{"storeDetails":"true","additionalData.card.encrypted.json":"adyenan0_1_1$<SECRET>
D/PaymentStateHandler: processPayment(): onSuccess
D/PaymentStateHandler: onStateChanged: PROCESSED
D/PaymentStateHandler: Payment processed.
D/PaymentStateHandler: Notifying the payment result to the merchant application
D/InAppPayment: paymentRequestListener.onPaymentResult()
```

We are going to workaround this by disabling the Adyen SDK for Android <= 19 versions (fallback to HPP).

**Comment by timon-langlotz** [comment]
Hi @tjerkw,

Did you also get a refused payment before on test, or were you unable to make any calls? A refused payment looks more like a different issue. It would be great if you could check with support (support@adyen.com) why the payment was refused.

Kind regards,
Timon

**Comment by tjerkw** [comment]
On Test I got an SSL (i think the app crashed).

**Comment by timon-langlotz** [comment]
Hi @tjerkw,

could you please still drop us a message on support@adyen.com? We would really like to figure this out. Would be great if you could include some data to identify the refused payment, e.g. the psp reference, shopper email, timestamp, etc.

Thanks in advance,
Timon

**Comment by timon-langlotz** [comment]
Hi @tjerkw,

we've just released 1.10.0, which has a fix for the SSL error.
Could you let us know if this resolves your issue?

Thanks,
Timon

**Comment by tjerkw** [comment]
Thanks @timon-langlotz ! 

We are thinking of support only android v19 and up, since this is also what android-pay (compile 'com.adyen.checkout:androidpay:1.11.0')  requires
